### PR TITLE
console: fix relationship table first row styling issue (closes #4974)

### LINKED ISSUE #4974: Relationships table 1st column overflows if the text is too long
Relationships table 1st column overflows if the text is too long as shown below.

image

### PR DESCRIPTION
**Pull request by kolharsam**
fixes #4974 

### Description
as mentioned in the issue, we have text on the first row of the relationship table overflowing if it exceeds the width of the row. This PR hopes to resolve this issue by placing text that overflows on the next line

### Changelog

- [x] Changelog is necessary since it is a user-facing change

### Affected components
- [x] Console

### Breaking Changes
 - [x] No Breaking Changes

### CODE REVIEW

**Comment by kolharsam** [comment]
> Can we have the same functionality as when adding a new relationship to be consistent? (overflow scroll)
> 2020-06-03 at 22 35 02 - Moccasin Tiger

updated!

**Comment by kolharsam** [comment]
@beerose I think your remarks have been addressed. Is there anything I missed?

File: CHANGELOG.md [hunk 1]
```diff
@@ -131,6 +133,8 @@ hasura seed apply --file 1234_add_some_seed_data.sql
 - server: fix importing of allow list query from metadata (fix #4687)
 - server: flush log buffer during shutdown (#4800)
 - server: fix edge case with printing logs on startup failure (fix #4772)
+- console: provide option to cascade metadata on dependency conflicts on console (fix #1593)
+- console: fix enum tables reload data button UI (#4647)
```
Comment (beerose) [inline CHANGELOG.md:137]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,9 +10,11 @@
 - server: fix introspection when multiple actions defined with Postgres scalar types (fix #5166) (#5173)
 - console: allow manual edit of column types and handle array data types (close #2544, #3335, #2583) (#4546)
 - console: add the ability to delete a role in permissions summary page (close #3353) (#4987)
+- console: fix styling of table row contents on tables on relationship page (#4974)
 - cli: handle missing files during metadata apply (close #5163) (#5170)
 - docs: add page on Relay schema (close #4912) (#5150)
 
+
```
Comment (beerose) [inline CHANGELOG.md:17]:
```suggestion
```

**Pull request merged**